src/test/mir-opt/const_allocation/rustc.main.ConstProp.after.mir

```diff
@@ -26,43 +26,43 @@ fn  main() -> () {
     }
 }
 
-alloc0 (const_allocation[317d]::FOO[0]) size: 16, align: 8 {
-    ├──────alloc17+0──────┤ 03 00 00 00 00 00 00 00 
+alloc0 (static: const_allocation[317d]::FOO[0], size: 16, align: 8) {
+     ├──────alloc17+0──────┤ 03 00 00 00 00 00 00 00
 }
 
-alloc17 size: 72, align: 8 {
-    0x00 │ 00 00 00 00 __ __ __ __ ├──────alloc4+0───────┤ 00 00 00 00 00 00 00 00 00 00 00 00 __ __ __ __ 
-    0x20 │ ├──────alloc8+0───────┤ 02 00 00 00 00 00 00 00 01 00 00 00 2a 00 00 00 ├──────alloc13+0──────┤ 
-    0x40 │ 03 00 00 00 00 00 00 00 
+alloc17 (size: 72, align: 8) {
+    0x00 │ 00 00 00 00 __ __ __ __ ├──────alloc4+0───────┤ 00 00 00 00 00 00 00 00 00 00 00 00 __ __ __ __
+    0x20 │ ├──────alloc8+0───────┤ 02 00 00 00 00 00 00 00 01 00 00 00 2a 00 00 00 ├──────alloc13+0──────┤
+    0x40 │ 03 00 00 00 00 00 00 00
 }
 
-alloc13 size: 48, align: 8 {
-    0x00 │ ├──────alloc12+0──────┤ 03 00 00 00 00 00 00 00 ├──────alloc14+0──────┤ 03 00 00 00 00 00 00 00 
-    0x20 │ ├──────alloc15+0──────┤ 04 00 00 00 00 00 00 00 
+alloc13 (size: 48, align: 8) {
+    0x00 │ ├──────alloc12+0──────┤ 03 00 00 00 00 00 00 00 ├──────alloc14+0──────┤ 03 00 00 00 00 00 00 00
+    0x20 │ ├──────alloc15+0──────┤ 04 00 00 00 00 00 00 00
 }
```

Comment:
Looking at this makes me wish the `├─...─┤` could be more "convex", i.e. `(─...─)` but that looks slightly worse because it doesn't connect.

I think the use of vertical bars for offsets is fine, but it makes `├─...─┤` look even more out of place. I just wish I had a good suggestion for what to replace the latter with.

Reply:
I think `⧃` could work, but there are no other rotations of that?! wth unicode?

Reply:
How do you like the latest version?

```
alloc24 (size: 72, align: 8) {
    0x00 │ 00 00 00 00 __ __ __ __ ╾──────alloc9+0───────╼
    0x10 │ 00 00 00 00 00 00 00 00 00 00 00 00 __ __ __ __
    0x20 │ ╾──────alloc14+0──────╼ 02 00 00 00 00 00 00 00
    0x30 │ 01 00 00 00 2a 00 00 00 ╾──────alloc22+0──────╼
    0x40 │ 03 00 00 00 00 00 00 00
}
```

Looks better in vscode -.-

Reply:
Yeah the GH font has a weird bump in the lines there.
But my local shell font is fine with it.^^ Seems okay, overall.